usv_ahc_py/cluster clust
sensor_msgs/CameraInfo cam_info
---
geometry_msgs/Point32 root_point
geometry_msgs/Point32 width_height
